lib/charms/grafana_agent/v0/cos_agent.py

```diff
@@ -641,7 +605,16 @@ def _remote_data(self) -> List[CosAgentProviderUnitData]:
                 continue
             if not (provider_data := self._validated_provider_data(raw)):
                 continue
-            all_data.append(provider_data)
+
+            # Apply topology
```

Comment:
unclear comment. How is instantiating this object 'applying' the topology?

Reply:
or appending it to a list?

Reply:
Yeah this was a comment from a previous implementation, removing.